Пример не запускается.

Сделал пустой стек проект, добавил lxcops как зависимость, сделал Main.hs 
как в примере там появляются ошибки. 

Было бы хорошо чтобы Main.hs с текстом из примера запускался. 

~~~
/home/anton/x/lxc/app/Main.hs:12:1: error:
    Parse error: naked expression at top level
    Perhaps you intended to use TemplateHaskell
~~~

Там не хватает main, но потом он ругается, что не хватает Applicative, 
лучше всё же рабочий пример иметь.

--- a/examples/lxcops-example/LICENSE
+++ b/examples/lxcops-example/LICENSE
@@ -1,0 +1,30 @@
+Copyright Author name here (c) 2017
+
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+
+    * Redistributions in binary form must reproduce the above
+      copyright notice, this list of conditions and the following
+      disclaimer in the documentation and/or other materials provided
+      with the distribution.
+
+    * Neither the name of Author name here nor the names of other
+      contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+"AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.

--- a/examples/lxcops-example/README.md
+++ b/examples/lxcops-example/README.md
@@ -1,0 +1,1 @@
+# lxcops-example

--- a/examples/lxcops-example/Setup.hs
+++ b/examples/lxcops-example/Setup.hs
@@ -1,0 +1,2 @@
+import Distribution.Simple
+main = defaultMain

--- a/examples/lxcops-example/lxcops-example.cabal
+++ b/examples/lxcops-example/lxcops-example.cabal
@@ -1,0 +1,21 @@
+name:                lxcops-example
+version:             0.1.0.0
+-- synopsis:
+-- description:
+homepage:            https://github.com/githubuser/lxcops-example#readme
+license:             BSD3
+license-file:        LICENSE
+author:              Author name here
+maintainer:          example@example.com
+copyright:           2017 Author name here
+category:            Web
+build-type:          Simple
+cabal-version:       >=1.10
+extra-source-files:  README.md
+
+executable lxcops-example
+  hs-source-dirs:      src
+  main-is:             Main.hs
+  default-language:    Haskell2010
+  build-depends:       base >= 4.7 && < 5
+                     , lxcops

--- a/examples/lxcops-example/src/Main.hs
+++ b/examples/lxcops-example/src/Main.hs
@@ -1,16 +1,11 @@
-# lxcops
-
-Это обёртка над консольными командами lxc.
-Интерфейс — как в библиотеке https://hackage.haskell.org/package/lxc, но без
-использования `bindings-lxc`, и реализован только частично, по необходимости.
-
-#### Пример использования
-
-[Main.hs](./examples/lxcops-example/src/Main.hs)
-
-
-```haskell
 module Main where
+
+-- Compile and run this:
+-- ```
+-- stack build
+-- stack install
+-- sudo lxcops-example
+-- ```
 
 import Data.Monoid
 import Control.Applicative
@@ -52,4 +47,3 @@ main = do
   withContainer (Container "new-name" Nothing) $ do
       ci <- containerInfo
       liftIO $ putStrLn $ "Clone: " <> show ci
-```

--- a/examples/lxcops-example/stack.yaml
+++ b/examples/lxcops-example/stack.yaml
@@ -1,0 +1,20 @@
+resolver: lts-9.17
+packages:
+- .
+- location:
+    git: https://github.com/hexresearch/lxcops
+    commit: 61991cc208e3005f058c25bdaa0ae0533913b623
+  extra-dep: true
+extra-deps:
+- HSH-2.1.3
+- transformers-free-1.0.1
+flags: {}
+extra-package-dbs: []
+nix:
+  enable: false
+  packages:
+    - binutils
+    - git
+    - gmp
+    - openssl
+    - zlib

